R/rd-include-rmd.R

```diff
@@ -0,0 +1,61 @@
+
+block_include_rmd <- function(tag, block, base_path) {
+  rmd <- tag$val
+  stopifnot(is.character(rmd), length(rmd) == 1, !is.na(rmd))
+  if (!requireNamespace("rmarkdown", quietly = TRUE)) {
```

Comment:
`is_installed()` might be a little cleaner since we're using rlang already

Reply:
OK. I also moved this check a bit upstream.